data/translation/Zed/de_DE.csv

```diff
@@ -72,3 +72,4 @@ Connections,Verbindungen
 Line,Zeile
 "You can’t use ""%totals_mode%"" for ""%connection_format%"" connection - please choose another one.","Sie können die Verbindung ""%totals_mode%"" for ""%connection_format%"" nicht verwenden - wählen Sie bitte eine andere Verbindung aus."
```

Comment:
```suggestion
"You can’t use ""%totals_mode%"" for ""%connection_format%"" connection - please choose another one.","Sie können die Verbindung ""%totals_mode%"" für ""%connection_format%"" nicht verwenden - wählen Sie bitte eine andere Verbindung aus."
```

But if `%totals_mode%` and `%connection_format%` is what I think it is, then the whole sentence should be differently worded:

> Sie können den Modus %totals_mode% für die Verbindung %connection_format% nicht verwenden, bitte verwenden Sie eine andere Verbindung.

Reply:
that requires approve from PO/TW